---
output: 
  rmarkdown::html_document:
    theme: flatly
---


# Motivation

***
## Microorganisms are incredibly diverse, and this diversity is also reflected in their metabolic capabilities (Hug et al. 2016). The RbiMs shiny app is designed to help visualize the metabolic pathways reconstructed through KEGG.


# Uploading data into Rbims

***
```{css, echo=FALSE}

body {
  font-size: 18pt;
  }
  

```
1. Uploading data using KofamKoala in KEGG
 
+ Prior to uploading your data to RbiMs there are some things we must do to prepare the data. 
 
+ First, download your assembly in a FASTA file. 
  + If you're uploading a file from NCBI, please follow the NCBI tutorial below.
+ Upload .faa files to KEGG in KofamKOALA
  + https://www.genome.jp/tools/kofamkoala/
+ Once KEGG data is obtained, we need to change the gene names to indicate bins and scaffolds
+ We do this by putting every row in gene name in the following format:
  + Bin_xx-scaffold_xx or Bin_xx_scaffold_xx
+ Now the data is ready to upload in the KEGG tab in the Upload data section
2. Uploading data using Interpro
  
</font>


```{r}

```


